src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/data/VisitResponse.kt

```diff
@@ -126,9 +127,19 @@ data class VisitResponse(
     visitorConcernText = visitEntity.visitorConcernText,
     visitors = visitEntity.visitors.filter { visitor -> visitor.person != null }
       .map { visitor -> Visitor(visitor.person!!.id, visitor.groupLeader) },
-    visitOutcome = null,
-    leadVisitor = null
+    visitOutcome = null, // TODO
+    leadVisitor = visitEntity.visitors.find { visitor -> visitor.groupLeader }?.person?.let {
+      LeadVisitor(
+        personId = it.id,
+        fullName = "${it.firstName} ${it.lastName}",
+        telephones = it.phones.toTelephoneList() + it.addresses.flatMap { address -> address.phones.toTelephoneList() }
+      )
+    }
   )
 
   data class CodeDescription(val code: String, val description: String)
 }
+
+fun List<Phone>.toTelephoneList(): List<String> {
+  return this.map { phone -> phone.phoneNo ?: "" }
```

Comment:
this looks a bit odd to me, wouldn't it be better to do a `mapNonNull` rather than end up with empty phone numbers in the list?